beetsplug/bpd/__init__.py

```diff
@@ -640,8 +640,8 @@ class Command(object):
     """A command issued by the client for processing by the server.
     """
 
-    command_re = re.compile(br'^([^ \t]+)[ \t]*')
-    arg_re = re.compile(br'"((?:\\"|[^"])+)"|([^ \t"]+)')
+    command_re = re.compile(r'^([^ \t]+)[ \t]*')
+    arg_re = re.compile(r'"((?:\\"|[^"])+)"|([^ \t"]+)')
```

Comment:
Got it; thanks for clarifying—I assumed the commands were bytes.

It looks at first glance to me like the commands are Unicode on Python 2 also, right? If so, perhaps these changes are correct as-is?